core/src/main/resources/jenkins/agents/CloudSet/index.properties

```
@@ -1,5 +1,5 @@
 learnMoreDistributedBuilds=Learn more about distributed builds
 noCloudAvailable=There are no clouds currently setup, create one or install a plugin for more cloud options.
-noCloudPlugin=There are no cloud implementations for dynamically allocated agents installed.
+noCloudPlugin=There are no cloud implementation for dynamically allocated agents installed.
```

this isn't right, are is a [linking verb](https://www.yourdictionary.com/articles/is-are-sentences) and is plural.
while implementation is singular.

the existing text is bit developer specific really and I would say not user facing text, maybe this would be more appropriate (it would depend on what it looks like in place and any surrounding text I haven't reviewed that):

```suggestion
noCloudPlugin=There is no plugin installed that supports clouds.
```

---

Much better indeed.